src/credential/trustmark/__tests__/get-credential-trustmark.test.ts

```diff
@@ -0,0 +1,28 @@
+import { decode } from "@pagopa/io-react-native-jwt";
+import { getCredentialTrustmarkJwt } from "../get-credential-trustmark";
+import { createCryptoContextFor } from "../../../utils/crypto";
+import { generate } from "@pagopa/io-react-native-crypto";
+
+describe("getCredentialTrustmarkJwt", () => {
+  it("should generate a JWT with the correct payload", async () => {
+    const ephemeralKeytag = `ephemeral-${Math.random()}`;
+    await generate(ephemeralKeytag);
+
+    const walletInstanceAttestation = "walletInstanceAttestation";
+    const wiaCryptoContext = createCryptoContextFor(ephemeralKeytag);
+    const credentialType = "MDL";
+    const documentNumber = "1234567890";
+
+    const jwt = await getCredentialTrustmarkJwt(
```

Comment:
You have a WIA decode in `getCredentialTrustmarkJwt`. I don't think it can use "walletInstanceAttestation" as a mocked value

Reply:
Here we go! https://github.com/pagopa/io-react-native-wallet/pull/156/commits/3e46f0cd86bb316d0345ee65b00dffac21695ccd